src/main/flight/interpolated_setpoint.c

```diff
@@ -58,9 +77,36 @@ FAST_CODE_NOINLINE float interpolatedSpApply(int axis, bool newRcFrame, ffInterp
         const float rxInterval = currentRxRefreshRate * 1e-6f;
         const float rxRate = 1.0f / rxInterval;
 
-        const float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
-        const float setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]) * pidGetDT();
+        float setpointSpeed = (rawSetpoint - prevRawSetpoint[axis]) * rxRate;
+        float setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]);
+        const float holdSteps = 2.0f;
+
+        // Glitch reduction code for identical packets
+        if ((setpointSpeed == 0) && (fabsf(rawSetpoint) < 0.98f * ffMaxRate[axis])) {
+            // identical packets, not at full deflection
+            if ((holdCount[axis] < holdSteps) && (fabsf(rawSetpoint) > 2.0f) && (bigStep[axis] == 0)){
```

Comment:
Missing whitespace. :space_invader:

Reply:
sorry can't figure where you mean - happy to fix, just not sure where

Reply:
ah, thanks Miguel - fixed